include/openthread/channel_manager.h

```diff
@@ -47,21 +47,28 @@ extern "C" {
  * @brief
  *   This module includes functions for Channel Manager.
  *
- *   The functions in this module are available when Channel Manager feature
- *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) is enabled. Channel Manager is available only on an FTD build.
+ *   The functions in this module are available when Channel Manager features
+ *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) and (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`) is
+ *   enabled. Channel Manager behavior depends on the device role. On a Full Thread Device (FTD) manages the
```

Comment:
FTD, MTD refers to the compile-time capability while SED, SSED are run-time roles. I think the precise description here should be `On a non-SSED device`.